style.css

```diff
@@ -1,455 +1,1108 @@
+  width: 100%;
+  background-color: #fff;
+  background-image: url(img/header-shapes\ mobile@2x.svg);
+  background-repeat: no-repeat;
+  top: 2.875rem;
+  border-bottom-left-radius: 60px;
+  height: 44.88rem;
+  margin-left: 6px;
 }
 
-h2 {
-  width: 327px;
-  height: 150px;
+.greeting-hearder {
+  display: flex;
   font-family: "Poppins", sans-serif;
   font-style: normal;
-  font-weight: 700;
   font-size: 40px;
+  font-weight: 700;
   line-height: 52px;
-  display: flex;
   align-items: center;
+  width: 327px;
+  height: 9.37rem;
+  margin: 20px;
   color: #172b4d;
-  flex: none;
   order: 0;
-  flex-grow: 0;
-  margin-top: -3rem;
-  margin-bottom: -3rem;
 }
 
-.container p {
-  width: 327px;
-  height: 146px;
+.greeting-text {
   font-family: "Poppins", sans-serif;
   font-style: normal;
+  width: 327px;
+  height: 146px;
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
+  margin: 0 20px;
   color: #344563;
-  flex: none;
   order: 1;
-  flex-grow: 0;
-  left: 24px;
-  top: 276px;
 }
 
-.Logo {
-  width: 76px;
-  height: 20px;
+.connect {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
   font-family: "Poppins", sans-serif;
   font-style: normal;
-  font-weight: 700;
-  font-size: 18px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
-  text-align: center;
-  color: #6070ff;
-  flex: none;
-  order: 0;
+  text-decoration: none;
+  font-size: 16px;
+  font-weight: 500;
+  margin: 0 20px;
+  order: 2;
+  line-height: 24px;
+  color: #7f8cff;
   flex-grow: 0;
-  position: absolute;
-  top: 14px;
-  left: 10px;
+  overflow-x: hidden;
 }
 
-.Union {
-  position: absolute;
-  top: 14px;
-  right: 10px;
+.twitter:hover,
+.linkedin:hover,
+.github:hover,
+.medium:hover,
+.angel:hover {
+  color: #505f79;
+  opacity: 0.5;
 }
 
-.social-icons {
-  list-style: none;
-  padding: 0;
+#social-icons {
   display: flex;
-  width: 170.17px;
-  height: 20px;
-  order: 3;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex: none;
   flex-grow: 0;
-  margin-left: 0.2rem;
-}
-
-.social-icons li {
-  margin-right: 10px;
+  list-style: none;
+  width: 10.636rem;
+  margin-left: -25px;
+  gap: 8px;
+  order: 3;
 }
 
-.social-icons li:last-child {
-  margin-right: 0;
+#social-icons li:hover {
+  transform: scale(1.5);
+  transition: 0.5s;
 }
 
-#my-section {
-  background-image: url(img/header-shapes\ mobile@2x.svg);
-  background-size: cover;
-  background-position: center;
-  width: 381px;
+/* ----The Work Section */
+.work {
+  background: #e5e5e5;
+  display: grid;
+  grid-template-columns: 33.333%;
+  margin-top: 6rem;
 }
 
-header {
-  margin-top: -3rem;
+@media only screen and (min-width: 375px) {
+  .card-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 16px;
+    gap: 12px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    width: 20rem;
+    height: 552px;
+    order: 0;
+    flex-grow: 0;
+    margin: 6px;
```

Comment:
- [ ] According to the previouse reviewer which say [see here](https://rawcdn.githack.com/Edgahkipkemoi/mobile/0229dbf05b3dc9b4199db0930f688d1dcd261741/index.html) Kindly adhere to the project requirement as even mentioned by your previous code reviewer regarding your design not matching properly with the Figma design. Your design should start responsive from the 768px screen size , I would suggest you to please don't copy the styles from Figma. Try to take just the design as the reference to build with the help of CSS. Right now because as I can see you are not using the Correct layout, media-query etc. which makes your design inconsistent and looks unprofessional. Please have a look at the below-attached screen recording.
Requirement


- [ ] see screen rec 

https://github.com/Edgahkipkemoi/mobile/assets/92458236/7b91135a-b338-4fe0-9b21-739676fb05d1